controllers/profiles.js

```diff
@@ -0,0 +1,24 @@
+// const User = require("../models/user");
+
+const ProfilePage = {
+  Index: (req, res) => {
+    let currentUsername = null;
+    if (req.session.user) currentUsername = req.session.user.username;
+    res.render("profiles/index", {
+      profileUsername: req.params.username,
+      currentUsername: currentUsername,
+    });
```

Comment:
Does the if statement say "if a user is logged in, then..."? And what is the difference between profileUsername and currentUsername?